backend/src/main.ts

```diff
@@ -53,20 +54,25 @@ const run = async () => {
   const queue = queueServiceFactory(appCfg.REDIS_URL);
   const keyStore = keyStoreFactory(appCfg.REDIS_URL);
 
-  const server = await main({ db, auditLogDb, smtp, logger, queue, keyStore });
+  const pkcs11Module = initializePkcs11Module();
```

Comment:
If its used and passed around as hsmModule why not call it like that? Felt confusing